src/Violation/Renderer/Html/HtmlRenderer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SensioLabs\DeprecationDetector\Violation\Renderer\Html;
+
+use SensioLabs\DeprecationDetector\Violation\Renderer\MessageHelper\MessageHelper;
+use SensioLabs\DeprecationDetector\Violation\Renderer\RendererInterface;
+use SensioLabs\DeprecationDetector\Violation\Violation;
+use Symfony\Component\Filesystem\Filesystem;
+
+class HtmlRenderer implements RendererInterface
```

Comment:
IMO `Renderer` as class name is better, because it already is in the Html namespace